src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -241,19 +253,215 @@ uct_ib_mlx5_devx_reg_ksm_data(uct_ib_mlx5_md_t *md,
     return status;
 }
 
+static ucs_status_t
+uct_ib_mlx5_devx_reg_ksm_data_contig(uct_ib_mlx5_md_t *md, uct_ib_mlx5_mr_t *mr,
+                                     off_t off, struct mlx5dv_devx_obj **mr_p,
+                                     uint32_t *mkey)
+{
+    const size_t reg_length = UCT_IB_MD_MAX_MR_SIZE;
```

Comment:
just use `UCT_IB_MD_MAX_MR_SIZE` directly, no need to declare `reg_length`

Reply:
👍